source/includes/fundamentals/code-snippets/VersionedApiExample.java

```diff
@@ -0,0 +1,41 @@
+package fundamentals;
+
+import java.util.Arrays;
+
+import com.mongodb.MongoClientSettings;
+import com.mongodb.ServerAddress;
+import com.mongodb.ServerApi;
+import com.mongodb.ServerApiVersion;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+
+public class VersionedAPIExample {
+
+    private static void setApiVersionOptions(String uri) {
+        // start apiVersionOptions
+        ServerApi serverApi = ServerApi.builder()
+                .version(ServerApiVersion.V1)
+                .strict(true)
+                .deprecationErrors(true)
+                .build();
+        // end apiVersionOptions
+    }
+
+    public static void main(String[] args) {
+        // start serverAPIVersion
+        ServerApi serverApi = ServerApi.builder()
+                .version(ServerApiVersion.V1)
+                .build();
+
+        ServerAddress serverAddress = new ServerAddress("localhost", 27017);
```

Comment:
I think it's more common to use a connection string and use MongoClientSetting.Builder.applyConnectionString